README.md

```diff
@@ -86,74 +80,214 @@ import (
     "fmt"
```

Comment:
Can't comment on lines not included but up above on line 67: If I'm understanding correctly, I might recommend slightly clarifying:

`Amongst all the peers in my set of N, am I the owner of the key "foo"? (e.g. does it consistent hash to 5?) If so, load it from source and store in local memory.`

If I misunderstood: don't mind me 😅